dot/network/config.go

```diff
@@ -94,6 +98,9 @@ type Config struct {
 	// privateKey the private key for the network p2p identity
 	privateKey crypto.PrivKey
 
+	// NodeKey is the private hex encoded Ed25519 key to build the p2p identity
+	NodeKey string
+
```

Comment:
supernit move this up above unexported fields (here `privateKey`) so it's grouped with exported fields?